src/schema/rules/files/raw/task.yaml

```diff
@@ -91,6 +91,16 @@ events__pet:
     reconstruction: optional
     run: optional
 
+events__mrs:
+  $ref: rules.files.raw.task.events
+  datatypes:
+    - mrs
+  entities:
+    $ref: rules.files.raw.task.events.entities
+    reconstruction: optional
```

Comment:
Isn't `reconstruction` a within-acquisition entity? Like you can have `rec-defaced` and `rec-refaced` versions of a file, but they come from the same acquisition, so you wouldn't have separate events files for each one.

Reply:
It's currently permitted for MRI. I don't have a strong commitment to this, though.